NEWS.md

```diff
@@ -1,3 +1,26 @@
-## [v1.7.1]
-- TBD
-- 
\ No newline at end of file
+## v2.0.0
+### Features
+- Get Instances
+- Get Authorities 
+- Get Authority Detail
+- Get Instance Detail
+- Get Inventory Holdings
+- Get Identifier Types
+- Get Classification Types
+- Get Contributor Types
+- Get Contributor Name Types
+- Get Locations
+- Get Service Points
+- Get Nature of Content Terms
+- Get Mode of Issuances
+- Get Instance Formats
+- Get Inventory Items
+- Get Instance Types
+- Get Inventory Note Types
+- Get Alternative Title Types
+- Get Institutions, Campsues, Libraries and Locations
+- Get Material Types
+- Get Source Record for Bib
+- Get Source Record for Authority
+- Download instance and authority data
+- Get Instance View
```

Comment:
Can we instead list the tickets that are part of this release rather than the features of the module which I think should be covered in the readme instead. I think this would be more in line with community releases. I know there are 4 or 5 in the JIRA that we could link to.

Reply:
Also think we need a release ticket in the JIRA like this: https://folio-org.atlassian.net/browse/EDGEUSERS-1

Reply:
The news formatting should be like this I think: https://github.com/folio-org/edge-users/pull/3/files